_studio/shared/umc/codec/h265_dec/src/umc_h265_frame.cpp

```diff
@@ -34,7 +34,6 @@ namespace UMC_HEVC_DECODER
 H265DecoderFrame::H265DecoderFrame(UMC::MemoryAllocator *pMemoryAllocator, Heap_Objects * pObjHeap)
     : H265DecYUVBufferPadded(pMemoryAllocator)
     , m_ErrorType(0)
-    , m_pSlicesInfo(0)
```

Comment:
Why do u leave [m_pSlicesInfo] unitialized?

Reply:
Because we initialize it below (line 56-57)
m_pSlicesInfo = new H265DecoderFrameInfo(this, m_pObjHeap);

Reply:
It is unnecessary, and potentially error prone optimization. I'd recommend to leave default here.

Ideally you shouldn't have such initializations in ctor body, better to have all such assignments in init list before the body

Reply:
Done.